### DISCUSSION TITLE
wtf? i wanna to know

### DISCUSSION THREAD

**Discussion by ddddjoker** [share121/inter-knot#2246] [其他]
怎么我的绳网这么卡

**Comment by CAONIMA-5418** [comment]
答：因为这个绳网建在外国网站

**Comment by gong177** [comment]
+1

**Comment by ShineStarAurora** [comment]
我都不卡嘞

**Comment by cpu644** [comment]
笑点解析：三个词还能有语法错误
不过这个绳网现在还在借github的服务器 不开加速器会卡。

**Comment by jianjue1** [comment]
能富强一下还是建议富强<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >
